tracks/python/exercises/phone-number/mentoring.md

```diff
@@ -0,0 +1,54 @@
+# Mentoring
+
+## Phone Number
+
+The problem asks you to write a `PhoneNumber` class.
+The constructor takes an input string that may or may not contain a valid number.
+The constructor needs to validate the number.
+The `pretty()` method needs to return a formatted number.
+
+## Reasonable Solution
+
+```python
+class Phone:
+    """Phone number validator."""
+
+    def __init__(self, number: str):
+        """Load and clean a number, validating it."""
+        # Ignore all non-digits.
+        digits = ''.join(i for i in number if i.isdigit())
+
+        # For an 11-digit with country code, drop the country code.
+        if len(digits) == 11 and digits.startswith('1'):
+            digits = digits[1:]
+
+        if len(digits) != 10:
+            raise ValueError('Invalid number')
```

Comment:
If not required perhaps "Invalid Phone Number" is informative, more so than just an invalid number, since it is more specific than that.  Still, only a reasonable solution, so no reason to change.

Reply:
The exercise has since been updated to require specific error messages. I will update this solution to match the new requirements.